### PR TITLE
chore: Ox_lib callbacks, rework of zones, removal of start zone

### PR DESCRIPTION
**Pull request by YvanCywan**
**Describe Pull request**
 - Removed start zone as it really isn't needed
 - Ox_lib callbacks
 - Changed grape zones from poly to box
 - QBCore export deprecated from client side

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes] (Be honest)
- Does your code fit the style guidelines? [yes]
- Does your PR fit the contribution guidelines? [yes]

### CODE REVIEW

**Comment by BerkieBb** [comment]
If the scope is more than one thing, you can nust remove the (), making it chore:

**Comment by BerkieBb** [comment]
Debug is turned to true and looping state bags is not good as it goes through a (de)serialization process each time you get the state bag

**Comment by YvanCywan** [comment]
> Debug is turned to true and looping state bags is not good as it goes through a (de)serialization process each time you get the state bag

I assume you are talking about this block here?
```lua
local function StartWineProcess()
    CreateThread(function()
        wineStarted = true
        while winetimer > 0 do
            winetimer = winetimer - 1
            Wait(1000)
		end
		wineStarted = false
		loadIngredients = false
		finishedWine = true
		winetimer = Config.wineTimer
    end)
end
```

**Comment by BerkieBb** [comment]
> 

I'm not sure which one as I'm on the phone but I saw some code having a state bag and a control press check under it

File: client.lua [hunk 1]
```diff
@@ -123,7 +123,6 @@ local function workWine()
 	end
 
 	lib.showTextUI(Lang:t("task.load_ingrediants"), {position = 'right'})
-	if LocalPlayer.state.invBusy then return end
 	if IsControlJustReleased(0, 38) and not LocalPlayer.state.invBusy then
```
Comment (BerkieBb) [inline client.lua]: Still looping a state bag

**Comment by BerkieBb** [comment]
And now fix linting issues

**Comment by BerkieBb** [comment]
Scopes also don't include file types as it's not the file that you're referencing but the scope that it is in, client is a scope for every file that is a client_script

File: server.lua [hunk 2]
```diff
@@ -6,38 +6,36 @@ RegisterNetEvent('qb-vineyard:server:getGrapes', function()
     Player.Functions.AddItem("grape", amount)
 end)
 
-QBCore.Functions.CreateCallback('qb-vineyard:server:loadIngredients', function(source, cb)
+lib.callback.register('qb-vineyard:server:loadIngredients', function(source)
 	local src = source
 	local Player = QBCore.Functions.GetPlayer(src)
-    local grape = Player.Functions.GetItemByName('grapejuice')
+    local grape = exports.ox_inventory:GetItem(src, 'grapejuice', nil, true)
 	if not Player.PlayerData.items then
```
Comment (Manason) [inline server.lua]: improper indent

File: server.lua [hunk 3]
```diff
@@ -6,38 +6,36 @@ RegisterNetEvent('qb-vineyard:server:getGrapes', function()
     Player.Functions.AddItem("grape", amount)
 end)
 
-QBCore.Functions.CreateCallback('qb-vineyard:server:loadIngredients', function(source, cb)
+lib.callback.register('qb-vineyard:server:loadIngredients', function(source)
 	local src = source
 	local Player = QBCore.Functions.GetPlayer(src)
```
Comment (Manason) [inline server.lua]: improper indentation

File: server.lua [hunk 4]
```diff
@@ -6,38 +6,36 @@ RegisterNetEvent('qb-vineyard:server:getGrapes', function()
...
-    if not grape or grape.amount < 16 then
+    if not grape or grape < 16 then
         TriggerClientEvent('ox_lib:notify', source, {type = 'error', description = Lang:t("error.invalid_items")})
-        cb(false)
+        return false
     end
     Player.Functions.RemoveItem("grape", 16, false)
-    cb(true)
+    return true
 end)
```
Comment (Manason) [inline server.lua:39]: grapeJuice and loadIngredients could have a common function extracted from them. Maybe out of scope for this PR.

**Review by Manason — APPROVED** [review]

File: server.lua (hunk 2 above)
Comment (YvanCywan) [reply server.lua]: Very weird, this does not show on my editor.

File: server.lua (hunk 4 above)
Comment (YvanCywan) [reply server.lua:39]: Yes there is a common function to extract here, but I will do it on another PR later. :)

**Review by Manason — APPROVED** [review]

**Pull request merged**